subsys/bluetooth/services/fast_pair/use_case/locator_tag/fp_adv.c

```diff
@@ -722,22 +720,49 @@ int app_fp_adv_disable(void)
 	fp_adv_rpa_suspension_cancel();
 
 	/* Suspend the requested advertising until the fp_adv module reinitializes. */
-	fp_adv_mode_set(APP_FP_ADV_MODE_OFF);
+	fp_adv_mode_set(BT_FAST_PAIR_LOCATOR_TAG_FP_ADV_MODE_OFF);
 
 	fp_adv_conn_clear();
 
 	err = fp_adv_set_teardown();
 	if (err) {
-		LOG_ERR("Fast Pair: fp_adv_set_teardown failed (err %d)", err);
+		LOG_ERR("Fast Pair Locator Tag FP Adv: fp_adv_set_teardown failed (err %d)", err);
```

Comment:
Should we keep `is_enabled=false` even if the advertising set teardown fails? It could lead to problems